lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -207,43 +207,50 @@ redistribute_policy:
     config_set_append: ' <state> redistribute <protocol> route-policy <policy>'
 
 #config in vrf context
+# TODO: fix these for XR
 route_target_both_auto:
```

Comment:
It doesn't hurt to explicitly specify something for Nexus, even if xr is excluded.